package.json

```diff
@@ -170,11 +170,18 @@
     "winston-papertrail": "~1.0.4"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
     "agendash": "~0.4.0",
+    "babel-loader": "^8.0.4",
     "concurrently": "~4.1.0",
+    "eslint-plugin-react": "^7.11.1",
     "exports-loader": "^0.7.0",
     "expose-loader": "^0.7.5",
     "faker": "~4.1.0",
+    "firebase": "^3.9.0",
```

Comment:
Let's add this back to `master` in a separate PR.

https://github.com/Trustroots/trustroots/pull/911#issuecomment-444243454